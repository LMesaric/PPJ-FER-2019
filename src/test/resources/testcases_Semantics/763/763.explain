semantic: a is not l-value
note: 'a = "abcd"' on line 22
