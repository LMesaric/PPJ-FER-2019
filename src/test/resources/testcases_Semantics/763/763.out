<izraz_pridruzivanja> ::= <postfiks_izraz> OP_PRIDRUZI(22,=) <izraz_pridruzivanja>
